composer.json

```diff
@@ -18,6 +35,9 @@
     "phpunit/phpunit": "~5.5",
     "symfony/config":  "2.3|~3.0"
   },
+  "scripts": {
+    "test": "phpunit tests/"
```

Comment:
After changing the `tests` directory name to `Tests` you can change this to `"test": "phpunit"`.